filter_plugins/oo_filters.py

```diff
@@ -18,9 +18,9 @@
 from distutils.util import strtobool
 from distutils.version import LooseVersion
 from operator import itemgetter
-from ansible.module_utils.six.moves.urllib.parse import urlparse
 from ansible.parsing.yaml.dumper import AnsibleDumper
 from six import string_types
+from six.moves.urllib.parse import urlparse
```

Comment:
Hmm, why? wouldn't that be one less point of variation and one less dependency to manage since we'd have to assume ansible is installed anyway?